.circleci/config.yml

```diff
@@ -75,19 +75,24 @@ jobs:
           command: |
             ./gradlew styled-recycler-view:githubRelease --stacktrace 2> /dev/null
             ./gradlew styled-switch-view:githubRelease --stacktrace 2> /dev/null
+            ./gradlew styled-slider-view:githubRelease --stacktrace 2> /dev/null
 
       - run:
           name: Publish to JCenter
           command: |
             ./gradlew styled-recycler-view:bintrayUpload --stacktrace 2> /dev/null
             ./gradlew styled-switch-view:bintrayUpload --stacktrace 2> /dev/null
+            ./gradlew styled-slider-view:bintrayUpload --stacktrace 2> /dev/null
```

Comment:
Side thoughts: I have seen some people using `--no-daemon --console=plain`. Do you know what these are used for? I don't, but I know I need to learn more about gradle

Reply:
Let's try using `--no-daemon` in terms of saving memory. https://docs.gradle.org/current/userguide/gradle_daemon.html